src/units/abilities.cpp

```diff
@@ -1359,9 +1359,9 @@ bool attack_type::get_special_ability_bool(const std::string& special, bool spec
 	return false;
 }
 
-bool attack_type::bool_ability(const std::string& special, bool special_id, bool special_tags) const
+bool attack_type::has_special_and_abilities(const std::string& special, bool special_id, bool special_tags) const
```

Comment:
Surely this should be called `has_special_or_ability`. The implementation clearly uses or.

Reply:
It's done

Reply:
If you want to go with plural it should be `has_specials_or_abilities`. To me it doesn't look like this is about multiple specials or abilities though. The input is a single string.